src/utils/src/ExponentialBackoffRetryStrategy.c

```diff
@@ -232,6 +232,19 @@ STATUS getExponentialBackoffRetryStrategyWaitTime(PRetryStrategy pRetryStrategy,
     return retStatus;
 }
 
+STATUS getExponentialBackoffRetryCount(PRetryStrategy pRetryStrategy, PUINT32 retryCount) {
+    ENTERS();
+    STATUS retStatus = STATUS_SUCCESS;
+    PExponentialBackoffRetryStrategyState pRetryState = NULL;
+    MUTEX_LOCK(pRetryState->retryStrategyLock);
```

Comment:
This will crash right.... you just set `pRetryState` to NULL and now you're deref'ing it is this a typo should it be `pRetryStrategy` instead?

Reply:
That's a good point. Divya, I think we can pass pRetryStrategy OR typecast pRetryState to KvsRetryStrategy (by making pRetryState as a first element of KvsRetryStrategy struct). Then we should validate KvsRetryStrategy->retryStrategyType if exponential backoff. And then only proceed.